<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fadccb;
font-family: Roboto;</string>
  </property>
  <widget class="QPushButton" name="eventsBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>109</width>
     <height>110</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>110</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{padding: 12px 0px 16px 0px;
  justify-content: center;
  align-items: center;
  gap: 4px;}
QPushButton:hover {
background: #FFFFFF;
}

QPushButton:pressed{
background: #FFEAE5;
}
</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/events_btn.png</normaloff>../img/events_btn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="calendarBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>109</width>
     <height>110</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>110</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{padding: 12px 0px 16px 0px;
  justify-content: center;
  align-items: center;
  gap: 4px;}
QPushButton:hover {
background: #FFF1ED;
}

QPushButton:pressed{
background: #FFEAE5;
}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/calendar_selected_btn.png</normaloff>../img/calendar_selected_btn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="topMenu">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>971</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0  50 0 0px;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="categoryText">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #281401;
padding-left: 50px;
</string>
      </property>
      <property name="text">
       <string>Календарь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>991</width>
     <height>701</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border-top-left-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="accountBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>109</width>
     <height>110</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>110</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{padding: 12px 0px 16px 0px;
  justify-content: center;
  align-items: center;
  gap: 4px;}
QPushButton:hover {
background: #FFF1ED;
}

QPushButton:pressed{
background: #FFEAE5;
}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/account_btn.png</normaloff>../img/account_btn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>481</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: #e77d66;
selection-gap: 10px;</string>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tasksBackground">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>371</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>282</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="ChosenDate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>dd/mm/yyyy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ToDoListLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Мероприятия:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="NoEventsLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текст</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="tasksFrame">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>260</y>
     <width>301</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="showEventButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20%;</string>
    </property>
    <property name="text">
     <string>Задача</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>0</number>
    </property>
    <property name="autoRepeatInterval">
     <number>0</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="showEventButton2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20%;</string>
    </property>
    <property name="text">
     <string>Задача</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showEventButton3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20%;</string>
    </property>
    <property name="text">
     <string>Задача</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showEventButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20%;</string>
    </property>
    <property name="text">
     <string>Задача</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showEventButton5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20%;</string>
    </property>
    <property name="text">
     <string>Задача</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addEventBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>590</y>
     <width>170</width>
     <height>49</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>170</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: #281401;
  position: relative;
  width: 100%;
  height: 100%;
  background-color: #ffffff;
  justify-content: center;
  align-items: center;
  border-radius: 10px;
  border: 3px solid #fadccb;
  gap: 20px;
  display: flex;
  padding: 5px 20px 5px 20px;
  border-radius: 20px;}
QPushButton:hover {
background: #FFF1ED;
}

QPushButton:pressed{
background: #FFEAE5;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/add_icon.png</normaloff>../img/add_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>tasksBackground</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>eventsBtn</zorder>
  <zorder>calendarBtn</zorder>
  <zorder>topMenu</zorder>
  <zorder>accountBtn</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>tasksFrame</zorder>
  <zorder>addEventBtn</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
